router-tests/normalization_test.go

```diff
@@ -11,7 +16,7 @@ import (
 )
 
 func TestNormalization(t *testing.T) {
-	t.Parallel()
+	//t.Parallel()
```

Comment:
Was this intended?

Reply:
Was testing if that was the issue. Reverting

Reply:
Done